typescript/infra/config/environments/mainnet3/agent.ts

```diff
@@ -613,6 +613,16 @@ const superswapIcaV2MatchingList = chainMapMatchingList({
   unichain: '0x43320f6B410322Bf5ca326a0DeAaa6a2FC5A021B',
 });
 
+// CTTP Token bridges that we use for rebalancing
```

Comment:
worth linking to registry config too
```suggestion
// CTTP Token bridges that we use for rebalancing
// https://github.com/hyperlane-xyz/hyperlane-registry/blob/main/deployments/warp_routes/USDC/mainnet-cctp-config.yaml
```